images/linux/scripts/installers/aws-sam-cli.sh

```diff
@@ -2,15 +2,31 @@
 ################################################################################
 ##  File:  aws-sam-cli.sh
 ##  Desc:  Installs AWS SAM CLI
-##         Must be run as non-root user after homebrew and clang
+##         Requires Pyton >=3.6, must be run as non-root user after toolset installation
 ################################################################################
 
 # Source the helpers for use with the script
 source $HELPER_SCRIPTS/document.sh
 
-# Install aws sam cli
-brew tap aws/tap
-brew install aws-sam-cli
+# Download latest aws sam cli sources
+TarballUrl=$(curl -s https://api.github.com/repos/awslabs/aws-sam-cli/releases/latest | jq -r '.tarball_url')
+TarballPath="/tmp/aws-sam-cli.tar.gz"
+wget $TarballUrl -O $TarballPath
+tar -xzvf $TarballPath -C /tmp
+SourcesDir=$(echo /tmp/awslabs-aws-sam-cli*)
```

Comment:
I think we can rid of SourcesDir variable and move cd from 23 line:
`cd /tmp/awslabs-aws-sam-cli*`